src/aat/resources/features/F-034/S-310.td.json

```diff
@@ -0,0 +1,32 @@
+{
+  "title": "must return 404 when case type does not exist",
+  "_guid_": "S-310",
+  "_extends_": "F-034_Test_Data_Base",
+
+  "specs": [
+    "an active profile in CCD",
+    "contains a non-existing case type id",
+    "contains a HTTP 404 Not Found"
+  ],
+
+  "request": {
+    "pathVariables": {
+      "ctid": "abc"
+    }
+  },
+
+  "expectedResponse": {
+    "_extends_": "Common_404_Response",
+    "body" : {
+      "exception" : "uk.gov.hmcts.ccd.endpoint.exceptions.ResourceNotFoundException",
+      "timestamp" : "[[ANY]]",
+      "status" : 404,
+      "error" : "Not Found",
```

Comment:
status and error are inherited from Common_404_Response

Reply:
Done